bin/wasm-node/rust/src/json_rpc_service.rs

```diff
@@ -44,6 +44,65 @@ use std::{
     sync::{atomic, Arc},
 };
 
+/// Spawns a task to handle incoming JSON-RPC requests.
+pub async fn request_handling_task(
+    mut tasks_executor: Arc<Mutex<Box<dyn FnMut(Pin<Box<dyn Future<Output = ()> + Send>>) + Send>>>,
+    clients: Arc<HashMap<usize, Arc<JsonRpcService>>>,
```

Comment:
```suggestion
    json_rpc_services: HashMap<usize, Arc<JsonRpcService>>,
```

Please create the `Arc` inside of the function's body, as it's required because of implementation details.